### PR TITLE
Add second order imex method

### PR DESCRIPTION
**Pull request by gardner48**
Add a default second order IMEX method (ARK2 from Giraldo, Kelly, and Constantinescu 2013)

### CODE REVIEW

**Review by drreynolds — DISMISSED** [review]
A couple of very minor changes, but otherwise this looks excellent!  I'll "approve" now.

**Comment by gardner48** [comment]
@balos1 I've added the "don't merge" label so can merge sundials-codes/answers#11 and update the submodule in this PR before merging.

**Review by balos1 — APPROVED** [review]

**Pull request merged**
